pages/domains/domains_choosing.md

```diff
@@ -42,7 +42,7 @@ Avoid using the following in your .gov domain name:
 - [Judicial branch federal agencies](#judicial-branch-federal-agencies)
 - [Legislative branch federal agencies](#legislative-branch-federal-agencies)
 - [Interstate organizations](#interstate-organizations)
-- [U.S. states and territories](#u.s.-states-and-territories)
+- [U.S. states and territories](#u-s-states-and-territories)
```

Comment:
I see your point on this! I think the intention behind this was so that screen readers could properly read U.S. as separate letters but can also see how this might introduce more complex anchor links -- will defer to @cisagov/gov-designers on this